test/index.html

```diff
@@ -183,6 +147,46 @@ <h1>Some Sample CFPB Charts</h1>
                      data-chart-type="bar">
                      This is the chart description.
                 </div>
+                <hr>
+                <h3>Indexed number of people with inquiries (auto)</h3>
+                <div class="cfpb-chart"
+                     data-chart-color="purple"
+                     data-chart-description="Indexed number of people with inquiries (auto)."
+                     data-chart-y-axis-label="Index (January 2009 = 100)"
+                     data-chart-source="consumer-credit-trends/auto-loans/inquiry_rate_AUT.csv"
+                     data-chart-type="line">
+                     This is the chart description.
+                </div>
+                <hr>
+                <h3>Indexed number of people with inquiries (credit cards)</h3>
+                <div class="cfpb-chart"
+                     data-chart-color="neutral"
```

Comment:
Switch this color to `purple`, since AFAIK all the index charts in every market will be Purple? Goes with the current scheme of all the origination charts across markets being Green, all the risk profile charts across markets being Pacific, etc.